data/scrapper/login.py

```diff
@@ -0,0 +1,35 @@
+import requests
+from bs4 import BeautifulSoup
+import getpass
+import time
+
+
+ERP_LOGIN_URL = 'https://erp.iitkgp.ac.in/IIT_ERP3/' #to get the session cookie
+SECURITY_QUESTION_URL = 'https://erp.iitkgp.ac.in/SSOAdministration/getSecurityQues.htm'
+
+
+def get_session_cookie ():
+    request_erp = requests.get(ERP_LOGIN_URL)
```

Comment:
Rather than using `requests` for making HTTP requests and `BeautifulSoup` for extracting data, we can use `requests-html` as a single interface for the entire scrapping process.

Refer: [kennethreitz/requests-html](https://github.com/kennethreitz/requests-html)